src/Tribe/Editor/Blocks/Event_Category.php

```diff
@@ -17,18 +17,31 @@ public function slug() {
 	 * Since we are dealing with a Dynamic type of Block we need a PHP method to render it
 	 *
 	 * @since 4.7
+	 * @since TBD - Added $classes variable to allow for the addition of custom CSS classes to the block output.
 	 *
 	 * @param  array $attributes
+	 * 
+	 * @var    string $classes The custom CSS classes to be added to the block output.
 	 *
-	 * @return string
+	 * @return string The rendered block output.
 	 */
 	public function render( $attributes = [] ) {
```

Comment:
Can we add a test around the `render()` to ensure the custom classes show when relevant?

Reply:
@stratease added tests